package.json

```diff
@@ -138,9 +138,9 @@
     "lint-staged": "lint-staged",
     "postinstall": "node ./scripts/linkPackages.js",
     "prepublishOnly": "node scripts/prepublishOnly.js",
-    "prettier": "prettier --config .prettierrc.js --write",
-    "prettier:check": "prettier \"**/**/*.{js,jsx,ts,tsx}\" --check --config .prettierrc.js",
-    "prettier:fix": "prettier \"**/**/*.{js,jsx,ts,tsx}\" --config .prettierrc.js --write",
+    "prettier": "prettier \"**/**/*.{js,jsx,ts,tsx}\" --config .prettierrc.js",
+    "prettier:check": "yarn prettier --check",
+    "prettier:fix": "yarn prettier --write",
```

Comment:
Looks good, please just use the command here as well:
![image](https://user-images.githubusercontent.com/5121148/93428922-937be100-f8c0-11ea-9ee0-f31e54acec95.png)